examples/Example3/common.cc

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+#ifdef __CUDACC__
```

Comment:
Maybe later on (i.e. in another PR). As I've mentioned in the description of this PR, this is an example developed initially outside of AdePT. It's being included here mainly for others to be able to refer to it later. For this reason, I think that it is useful to keep it in its original state at least at the point of inclusion into the repository. Later it can be migrated to use facilities from AdePT if it will be used as more than just as an example of how to handle particles on the GPU.